src/interfaces/special-resolution-interfaces/create-resolution-interface.ts

```diff
@@ -4,7 +4,7 @@ import { SigningPersonIF } from '@/interfaces'
 export interface CreateResolutionIF {
   resolutionConfirmed: boolean
```

Comment:
The question becomes in a way that once we submit "resolutionConfirmed" to BE, then next time loading the filing will use this property from BE. If we want to check for whether resolution form has been changed, then we can perhaps use a getter from the entitySnapshot and resolution properties, like "hasBusinessNameChanged". I'm not sure if that is the way to go, since we don't have special resolution for the entitySnapshot yet. I'm kind of confused as to what this property is actually doing.

Reply:
I can make this property as optional, I am not sure where we are taking entitySnapshot, If it's from draft , we can reset it from there. I think after above discussion we will have this value only if user clicks on save and resume later

Reply:
`this.parseSpecialResolutionFiling(filing, businessSnapshot)` is the one to set entity for specialResolution. We can add 'confirmed' to `ResolutionsIF` and initialize it like "hasBusinessNameChanged". We don't have to if the validation is handled inside the form. But when we click "File and Pay", if the confirmation checkbox is not checked, the UI should take us to the checkbox component.

Reply:
okey i will remove `resolutionConfirmed ` from this file and add it in `ResolutionsIF `

Reply:
This will make sure the schema is correct upon saving the draft. The question is how do we deal with the confirmation checkbox. I see there is another ticket for this issue, so we can address this issue in that ticket. I want to hear from other reviewers about this.

Reply:
My ears are burning, lol.

In other filings, we save some certify checkbox flags in the filing JSON even though the API doesn't use it.

In other filings, there is a requirement to NOT restore the checkbox when a draft is resumed. I know this filing is different.

As for where to store this flag, I think it's up to you. If you think it fits well in the Resolution object then use that. Some other filings save the flag in the header object. If it looks right / feels right then I'm OK with it :)

Reply:
Agreed. I withdraw the idea to use it in `ResolutionIF`. We can use this flag here and remove it from the payload.